src/i18n/DE/grid-resources.ts

```diff
@@ -0,0 +1,132 @@
+import { IGridResourceStrings } from 'igniteui-angular';
+
+// exported below as re-cast to create declaration type with expanded properties
+const GridResourceStringsDE_: ExpandRequire<IGridResourceStrings> = {
+  igx_grid_groupByArea_message: 'Lege eine Spaltenüberschrift hier ab, um danach zu sortieren.',
+  igx_grid_emptyFilteredGrid_message: 'Keine Einträge gefunden.',
+  igx_grid_emptyGrid_message: 'Grid hat keine Daten.',
+  igx_grid_filter: 'Filter',
+  igx_grid_filter_row_close: 'Schließen',
+  igx_grid_filter_row_reset: 'Zurücksetzen',
+  igx_grid_filter_row_placeholder: 'Füge Filterwert hinzu',
+  igx_grid_filter_row_boolean_placeholder: 'Alle',
+  igx_grid_filter_row_date_placeholder: 'Wähle Datum',
+  igx_grid_filter_operator_and: 'Und',
+  igx_grid_filter_operator_or: 'Oder',
+  igx_grid_complex_filter: 'Komplexer Filter',
+  igx_grid_filter_contains: 'Enthält',
+  igx_grid_filter_doesNotContain: 'Enthält nicht',
+  igx_grid_filter_startsWith: 'Beginnt mit',
+  igx_grid_filter_endsWith: 'Endet mit',
+  igx_grid_filter_equals: 'Gleich',
+  igx_grid_filter_doesNotEqual: 'Nicht gleich',
+  igx_grid_filter_empty: 'Leer',
+  igx_grid_filter_notEmpty: 'Nicht leer',
+  igx_grid_filter_null: 'Null',
+  igx_grid_filter_notNull: 'Nicht Null',
+  igx_grid_filter_before: 'Vor',
+  igx_grid_filter_after: 'Nach',
+  igx_grid_filter_today: 'Heute',
+  igx_grid_filter_yesterday: 'Gestern',
+  igx_grid_filter_thisMonth: 'Diesen Monat',
+  igx_grid_filter_lastMonth: 'Letzten Monat',
+  igx_grid_filter_nextMonth: 'Nächsten Monat',
+  igx_grid_filter_thisYear: 'Dieses Jahr',
+  igx_grid_filter_lastYear: 'Letztes Jahr',
+  igx_grid_filter_nextYear: 'Nächstes Jahr',
+  igx_grid_filter_greaterThan: 'Größer als',
+  igx_grid_filter_lessThan: 'Kleiner als',
+  igx_grid_filter_greaterThanOrEqualTo: 'Größer als oder gleich',
+  igx_grid_filter_lessThanOrEqualTo: 'Kleiner als oder gleich',
+  igx_grid_filter_true: 'Wahr',
+  igx_grid_filter_false: 'Falsche',
+  igx_grid_filter_all: 'Alle',
+  igx_grid_filter_condition_placeholder: 'Wähle Filter',
+  igx_grid_summary_count: 'Anzahl',
+  igx_grid_summary_min: 'Min',
+  igx_grid_summary_max: 'Max',
+  igx_grid_summary_sum: 'Summe',
+  igx_grid_summary_average: 'Mittel',
+  igx_grid_summary_earliest: 'Früheste',
+  igx_grid_summary_latest: 'Späteste',
```

Comment:
How is this used? This is gendered so it depends what the gender is that it applies to. For example, earliest month would be "Frühester Monat", as opposed to "Früheste Woche" (earliest week, notice the additional 'r' at the end).

Reply:
check [this samle ](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/summaries)and its rightmost column

Reply:
You can interpret it is "[Das] früheste [Datum]" or as "Frühestes [Datum]". I'm not sure which is better.